config/permissions.php

```diff
@@ -130,5 +130,5 @@
+  "admin" => array("/tasks.php", "/analysistracker-summary.php", "/xptracker-summary.php", "/iterationForm.php", "/moduleForm.php", "/storyForm.php", "/sectionForm.php", "/taskStoryForm.php", "/taskSectionForm.php", "/viewProjectDetails.php", "/viewUsers.php", "/viewUserDetails.php", "/viewWorkingHoursResultsReport.php", "/viewIteration.php", "/viewModule.php", "/viewStory.php", "/viewSection.php", "/services/createProjectsService.php", "/services/createTasksService.php", "/services/deleteProjectsService.php", "/services/deleteTasksService.php", "/services/getCustomerProjectsService.php", "/services/getExtraHoursReportService.php", "/services/getIterationCustomStoriesService.php", "/services/getOpenTaskStoriesService.php", "/services/getPendingHolidayHoursService.php", "/services/getProjectAnalysisTrackerTree.php", "/services/getProjectIterationsService.php", "/services/getProjectTrackerTree.php", "/services/getProjectTtypeReportService.php", "/services/getProjectUserCustomerReportJsonService.php", "/services/getProjectUserCustomerReportService.php", "/services/getProjectUserStoryReportService.php", "/services/getStoryCustomTaskStoriesService.php", "/services/getUserCustomersService.php", "/services/getUserProjectCustomerReportJsonService.php", "/services/getUsersProjectsReportService.php", "/services/getUserTasksService.php", "/services/setTasksJsonService.php", "/services/updateProjectsService.php", "/services/updateTasksService.php", "/services/updateTaskStoriesService.php", "/services/createTaskStoriesService.php", "/services/deleteTaskStoriesService.php", "/services/updateTaskSectionsService.php", "/services/createTaskSectionsService.php", "/services/deleteTaskSectionsService.php", "/services/getSectionCustomTaskSectionsService.php", "/services/getAllUsersService.php", "/services/updateUsersService.php", "/services/createUsersService.php", "/services/deleteUsersService.php", "/services/getUserHourCostHistoriesService.php", "/services/getUserAreaHistoriesService.php", "/services/getUserCityHistoriesService.php", "/services/getUserJourneyHistoriesService.php", "/services/createJourneyHistoriesService.php", "/services/createHourCostHistoriesService.php", "/services/createAreaHistoriesService.php", "/services/createCityHistoriesService.php", "/services/updateJourneyHistoriesService.php", "/services/updateHourCostHistoriesService.php", "/services/updateAreaHistoriesService.php", "/services/updateCityHistoriesService.php", "/services/deleteJourneyHistoriesService.php", "/services/deleteHourCostHistoriesService.php", "/services/deleteAreaHistoriesService.php", "/services/deleteCityHistoriesService.php", "/viewProjects.php", "/userTasksReport.php", "/services/getTasksFiltered.php"),
```

Comment:
Since we were not using this array at all, let's clear it completely and only add `"/userTasksReport.php", "/services/getTasksFiltered.php"`. Better if you add them with a comment saying "User tasks report" so we better know what it is for in the future.